### ISSUE TITLE
v0.20.4: registry sync regression: failed to copy aws config to tempfile for update

### ISSUE THREAD

**Issue by nocnokneo** [common-fate/granted#580]
Starting with v20.5.4 (and v0.20.5) I get "failed to copy aws config to tempfile for update":

```
❯ sudo tar -zxf ~/Downloads/granted_0.20.3_linux_x86_64.tar.gz -C /usr/local/bin/

❯ granted registry sync
[i] From gitlab.com:cytotronics/cyto
[i]  * branch              HEAD       -> FETCH_HEAD
[✔] Successfully synced registry cyto

❯ sudo tar -zxf ~/Downloads/granted_0.20.4_linux_x86_64.tar.gz -C /usr/local/bin/

❯ granted registry sync
[✘] failed to copy aws config to tempfile for update
```

**Comment by shwethaumashanker** [comment]
@nocnokneo Can you please check your `~/.aws/config` file to see if the sync went through successfully in v0.20.3? v0.20.4 has [added logic](https://github.com/common-fate/granted/commit/dfa6ad4001495155a9c1c579d807c768ab5c245f) for when the registry sync fails. Can you please share what your `~/.granted/config` looks like for further debugging?

**Comment by nocnokneo** [comment]
The sync seems to work fine with 0.20.3. The mtime of `~/.aws/config` shows that the file was touched and the contents are correct and working. `[granted_registry_start cyto] ... [granted_registry_end cyto]` with all the expected profile configs.

This is my `~/.granted/config`:

```ini
DefaultBrowser = "FIREFOX"
CustomBrowserPath = "/usr/bin/firefox"
CustomSSOBrowserPath = ""
Ordering = ""
ExportCredentialSuffix = ""

[ProfileRegistry]
  PrefixAllProfiles = false
  PrefixDuplicateProfiles = false

  [[ProfileRegistry.Registries]]
    name = "cyto"
    url = "git@gitlab.com:cytotronics/cyto.git"
    path = "infra/granted-registry"

```

**Comment by shwethaumashanker** [comment]
Can you please check if the fixes we have made to the granted registry sync [v0.20.6](https://github.com/common-fate/granted/releases/tag/v0.20.6) resolve your issues?

**Comment by webframp** [comment]
Also seeing this for a few team members

**Comment by shwethaumashanker** [comment]
@nocnokneo and @webframp Can you please check if you are still seeing this issue with [v0.20.6](https://github.com/common-fate/granted/releases/tag/v0.20.6), we have added some fixes to `granted registry sync`

**Comment by nocnokneo** [comment]
Fixed with v0.20.6 for me!

**Comment by nocnokneo** [comment]
Unfortunately v0.20.3 continues to be the only usable release for me due to #591 regression

**Comment by webframp** [comment]
This issue is fixed in v0.20.6 for me, but now getting the same as @nocnokneo. Similar EKS use case as #591
